ext/storage_sources/s3_store/s3_log_system.h

```diff
@@ -0,0 +1,31 @@
+#include <wiredtiger.h>
+#include <wiredtiger_ext.h>
+
+#include <aws/core/Core_EXPORTS.h>
+#include <aws/core/utils/logging/LogSystemInterface.h>
+#include <aws/core/utils/logging/LogLevel.h>
+
+#include <atomic>
+
+class s3_log_system: public Aws::Utils::Logging::LogSystemInterface {
```

Comment:
The class name should be `S3LogSystem`. Only the file name needs to be lowercase with underscores.